src/Forms/Form/Form.stories.mdx

```diff
@@ -57,10 +87,538 @@ const fields = useMemo(
+- Default Value: `ColorResult` - Object containing all the different variants for the color selected.
+
+### ColorResult (Type)
+* **hex** - `string`
+* **hsl** - `object`
+	* **a** - `number | undefined`
+	* **h** - `number`
+	* **s** - `number`
+	* **l** - `number`
+* **rgb** - `object`
+	* **a** - `number | undefined`
+	* **r** - `number`
+	* **g** - `number`
+	* **b** - `number`
+
+### Props (Input Settings)
+This component doesn't require any additional props.
+
+### How to use in a form?
+```ts
+const fields = useMemo(
+	() => 
+		[
+			//...other fields
+			{
+				//...all generic field props,
+				type: 'color',
+			},
+			//...other fields
+		],
+	[]
+);
+```
+
+
+## FormFieldDateField
+- It allows the user to view and pick dates from a calendar widget or manually type the date in the text field.
+- [**Playground**](/?path=/story/formfields-formfielddatefield--playground)
+- Default Value: `string` - Date in UTC format transformed to string.
+
+### Props (Input Settings)
+* **showTime** - `boolean` optional - When true a time field will appear next to the date field.
+
+### How to use in a form?
+```ts
+const fields = useMemo(
+	() => 
+		[
+			//...other fields
+			{
+				//...all generic field props,
+				type: 'date',
+				inputSettings: {
+					showTime: false,
+				}
+			},
+			//...other fields
+		],
+	[]
+);
+```
+
+## FormFieldRadioButton
+- A group of radio buttons allows users to select a single item from a list of possible options.
+- All possible options are exposed up front for comparison.
+- Users can only make a single selection from the list of possible options.
+- **Playground**: https://simpleviewinc.github.io/sv-mosaic/?path=/story/formfields-formfieldradio--playground
+
+#### Props (Input Settings)
+* **options** - `array` of `object` optional - Predefined set of mutually exclusive options.
+	* **label** - `string` required - Description of the option.
+	* **value** - `string` required - Value of the option, this is stored in the Form state.
+
+#### How to use in a form?
+```ts
+const fields = useMemo(
+		() =>
+			[
+				//...other fields
+				{
+					//...all generic field props,
+					type: "radio",
+					inputSettings: {
+						options: [
+							{
+								label: "Label 1",
+								value: "label_1",
+							},
+							{
+								label: "Label 2",
+								value: "label_2",
+							},
+							{
+								label: "Label 3",
+								value: "label_3",
+							},
+						]
+					},
+					helperText: 'Helper text',
+					instructionText: 'Instruction text'
+				}
+				//...other fields
+			] as FieldDef<FormFieldRadioDef>[],
+		[]
+	);
+```
+
+## FormFieldText
+- Text input fields are used to enter text information, numbers, e-mail addresses, or passwords. 
+- A text field consists of a label and a line with variations in width.
+- [**Playground**](/?path=/story/formfields-formfieldtext--playground)
+
+#### Props (Input Settings)
+
+* **htmlFor** - `string` optional - Specifies which form element a label is bound to.

```

Comment:
I think htmlFor should be removed as inputSetting from all components. Since it should match the name then I think it's better if we just completely remove htmlFor and just "inject" the name value internally in the Col.tsx file